Azure/communication-ui-library-ios
# [Feature] Updated participant grid layout for iPad

## Purpose

Figma UX: https://www.figma.com/file/72qJ03EOQExf12g5JTdWjQ/Web-UI-Library?node-id=1287%3A0
Updated participant grid layout for iPad, iPhone UX is unaffected 
* max grid number changed from 6 to 9 on iPad
* Grid layout changes per different participants and orientation. (detailed in Figma and video)

Note: There are a few scenarios that are de-scoped from this PR, please skip those scenarios
* n=7 scenario
* n=10,11,12 scenario

Video

https://user-images.githubusercontent.com/90668345/169420073-a0256063-af67-430f-a6b6-8e78305a4685.mov



## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code
* Join a call with 0 to more than 10 remote participants 
* check to see if the grid are as per Figma 


## What to Check
Verify that the following are valid
* check to see if the grid are as per Figma https://www.figma.com/file/72qJ03EOQExf12g5JTdWjQ/Web-UI-Library?node-id=1287%3A0

## Other Information
